Version Packages (#3994)
hashicorp/design-system: packages/components/CHANGELOG.md

<small class="doc-whats-new-changelog-metadata">[#4042](https://github.com/hashicorp/design-system/pull/4042)</small>

<div class="doc-whats-new-changelog-separator"></div>

`AdvancedTable` - Fixed issue where keyboard order did not match visual order after sorting the table.

<small class="doc-whats-new-changelog-metadata">[#4044](https://github.com/hashicorp/design-system/pull/4044)</small>

<div class="doc-whats-new-changelog-separator"></div>

`StepperNav` - Added scren reader text for step number

`StepperList` - Added scren reader text for step number

<small class="doc-whats-new-changelog-metadata">[#3993](https://github.com/hashicorp/design-system/pull/3993)</small>

<div class="doc-whats-new-changelog-separator"></div>

`AdvancedTable` - Fixed an issue where clicking a cell scrolled the page to center that cell

<small class="doc-whats-new-changelog-metadata">[#4017](https://github.com/hashicorp/design-system/pull/4017)</small>

<div class="doc-whats-new-changelog-separator"></div>

`AdvancedTable` - Updated reorder handle to not be visible on resize handle hover / focus

> `AdvancedTable` - Fixed an issue where the table header tooltip was not visible with the table empty state

Reviewer: ```suggestion
`AdvancedTable` - FIxed a bug where tooltips in the table header would be cut off behind the empty state message.
```

# @hashicorp/design-system-components

## 6.5.0

### Minor Changes

<!-- START components/segmented-group -->

> `SegmentedGroup` - Added `SuperSelect` and `SuperSelectMultiple` as yielded contextual components.

Reviewer: ```suggestion
`SegmentedGroup` - Added `SuperSelectSingle` and `SuperSelectMultiple` as yielded contextual components.
```

<div class="doc-whats-new-changelog-separator"></div>

### Patch Changes

<!-- START components/filter-bar -->

`FilterBar` - Added support for `ActionsGeneric` wrapping to new line

<!-- END -->

<small class="doc-whats-new-changelog-metadata">[#4034](https://github.com/hashicorp/design-system/pull/4034)</small>

<div class="doc-whats-new-changelog-separator"></div>

<!-- START components/table/advanced-table -->

`AdvancedTable` - Fixed bug where the bottom scroll indicator was always present if reordering was enabled.

<!-- END -->

<small class="doc-whats-new-changelog-metadata">[#4042](https://github.com/hashicorp/design-system/pull/4042)</small>

<div class="doc-whats-new-changelog-separator"></div>

<!-- START components/table/advanced-table -->

`AdvancedTable` - Fixed issue where keyboard order did not match visual order after sorting the table.

<!-- END -->

<small class="doc-whats-new-changelog-metadata">[#4044](https://github.com/hashicorp/design-system/pull/4044)</small>

<div class="doc-whats-new-changelog-separator"></div>

<!-- START components/stepper/nav -->

> `StepperNav` - Added scren reader text for step number

Reviewer: ```suggestion
`StepperNav` - Added screen reader text for step number
```

<!-- END -->

<!-- START components/stepper/list -->

> `StepperList` - Added scren reader text for step number

Reviewer: ```suggestion
`StepperList` - Added screen reader text for step number
```

<!-- END -->

<small class="doc-whats-new-changelog-metadata">[#3993](https://github.com/hashicorp/design-system/pull/3993)</small>

<div class="doc-whats-new-changelog-separator"></div>

<!-- START components/table/advanced-table -->

> `AdvancedTable` - Fixed an issue where clicking a cell scrolled the page to center that cell

Reviewer: Anyone have ideas to tweak this text to clarify what the issue was? (I'm not clear on it from the text description, personally.)

Reviewer 2: Fixed an issue where clicking a cell in the table would jump the whole page to center the cell within the viewport.

<!-- END -->

<small class="doc-whats-new-changelog-metadata">[#4017](https://github.com/hashicorp/design-system/pull/4017)</small>

<div class="doc-whats-new-changelog-separator"></div>

<!-- START components/table/advanced-table -->

> `AdvancedTable` - Updated reorder handle to not be visible on resize handle hover / focus

Reviewer: Any ideas to tweak this to improve clarity? To me it's a little too brief/concise to the point where it's difficult to grasp what it means.

Reviewer 2: Fixed a bug where the reorder handle would be visible when the resize handle was hovered.

^ Is this better?

Reviewer: That's clearer to me, thanks

<!-- END -->

<!-- START components/table/advanced-table -->

> `AdvancedTable` - Fixed an issue where the table header tooltip was not visible with the table empty state

Reviewer: This could use a bit of tweaking for clarity, I think.

Reviewer 2: FIxed a bug where tooltips in the table header would be cut off behind the empty state.

Reviewer: Better. "behind the empty state message" perhaps?

Reviewer 2: yeah that works!
